### PR TITLE
Commit2

### PR DESCRIPTION
**Pull request by veronaRub**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,44 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Lesson 8</title>
+        <link rel="icon" type="image/x-icon" href="https://cdn-icons-png.flaticon.com/512/119/119598.png">
+        <title>Lesson 7 - version 7.1</title>
```
Comment (RomanIlkiv) [inline index.html]: Why changed from Lesson 8 to Lesson 7?:)

File: index.html [hunk 2]
```diff
@@ -1,10 +1,44 @@
...
     <body>
-        <h1>Lesson 8</h1>
+        <h1>Veronica Rubliuk</h1>
+        <img src="img.jpg" width="210" height="260" alt="lesson_7_robot.dreams_rub_ver">
+        <h3>junior front-end developer</h3>
+        <div>Work Expirience</div>
+        <br>
+        <table>
+            <thead>
+                <tr class="thHead">
```
Comment (RomanIlkiv) [inline index.html]: please, don't use capital letters in the name of classes

File: index.html [hunk 3]
```diff
@@ -1,10 +1,44 @@
...
+                    <th colspan="1">Period</th>
+                    <th colspan="3">Position</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr style="height:50px" class="tdBody">
```
Comment (RomanIlkiv) [inline index.html]: write styles inside of html is very bad practice, please move it to .css file

File: index.html [hunk 4]
```diff
@@ -1,10 +1,44 @@
...
+                    <td>2017 - 2018</td>
+                    <td>Office manager at <a href="https://jobs.dou.ua/companies/tekrum/reviews/"target="_blank">Tekrum</a></td>
+                </tr>
+                <tr style="height:50px" class="tdBody">
+                    <td>2018 - 2021</td>
+                    <td>Administrative manager at <a href="https://jobs.dou.ua/companies/tekrum/reviews/"target="_blank">Tekrum</a></td>
+                </tr>
+                <tr style="height:50px" class="tdBody">
```
Comment (RomanIlkiv) [inline index.html]: the same here and in other places

File: index.html (hunk 2 above)
Comment (veronaRub) [reply index.html]: okey

File: index.html (hunk 3 above)
Comment (veronaRub) [reply index.html]: +

File: index.html (hunk 4 above)
Comment (veronaRub) [reply index.html]: +

File: index.html (hunk 2 above)
Comment (veronaRub) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (veronaRub) [reply index.html]: done

File: index.html (hunk 4 above)
Comment (veronaRub) [reply index.html]: done

**Review by RomanIlkiv — APPROVED** [review]

**Pull request merged**